.github/workflows/deploy-gh-pages.yml

```diff
@@ -4,7 +4,6 @@ name: Deploy VitePress site to Pages
 on:
   # Runs on pushes targeting the default branch
   push:
-    branches: ["main"]
```

Comment:
let's restrict the workflow to only starting if the changes are in the /docs folder : https://docs.github.com/en/actions/using-workflows/triggering-a-workflow#example-including-paths

Reply:
done